images/linux/scripts/installers/rust.sh

```diff
@@ -9,8 +9,8 @@ source $HELPER_SCRIPTS/document.sh
 
 set -e
 
-export RUSTUP_HOME=/usr/share/rust/.rustup
-export CARGO_HOME=/usr/share/rust/.cargo
+export RUSTUP_HOME=/etc/skel/rust/.rustup
```

Comment:
I think you should also remove logic at the end of this file
![image](https://user-images.githubusercontent.com/16715858/79775099-67255e00-833c-11ea-8447-9bcdde7da1d7.png)